pkg/checkpoint/process_test.go

```diff
@@ -170,8 +173,9 @@ func TestProcess(t *testing.T) {
 				},
 				"AA": {},
 			},
-			expectStop:        []string{"AA", "kube-system/pod-checkpointer"},
-			expectGraceRemove: []string{"AA", "kube-system/pod-checkpointer"},
+			expectStart:       []string{"kube-system/pod-checkpointer"},
+			expectStop:        []string{"BB"},
+			expectGraceRemove: []string{"AA", "BB", "kube-system/pod-checkpointer"},
 		},
 		{
 			desc:         "Inactive pod-checkpointer, no local parent, no api parent: should remove all",
```

Comment:
Is this case possible? For the pod checkpointer to run it will at least be in localRunning right?
But ok to leave as is.

Reply:
Strictly speaking this shouldn't be possible...